assets/src/application/ui/display/TimezoneTimeInfo/Trigger.tsx

```diff
@@ -8,19 +8,17 @@ interface TriggerProps {
 	onClick?: VoidFunction;
 }
 
-const Trigger = forwardRef<typeof IconButton, TriggerProps>(
-	({ tooltip, ...props }, ref) => (
-		<IconButton
-			borderless
-			color='white'
-			className='ee-timezone-info-btn'
-			icon={ () => <GlobalOutlined /> }
-			tooltip={ tooltip }
-			tooltipProps={ { placement: 'top' } }
-			{ ...props }
-			ref={ ref }
-		/>
-	)
-);
+const Trigger = forwardRef<typeof IconButton, TriggerProps>(({ tooltip, ...props }, ref) => (
+	<IconButton
+		{...props}
+		borderless
+		color='white'
+		className='ee-timezone-info__button'
+		icon={() => <GlobalOutlined className='ee-timezone-info__icon' />}
```

Comment:
I think it's better to lift these kind of styles at higher leve, had created `withSize` HOC, if any more props will be needed, can rename this differently.